### PR TITLE
[RPC] Make RPC standalone

### PR DESCRIPTION
**Pull request by mikedennis**
Add node DI services to RPC Feature DI services when configuring
the RPC feature. Technique taken from Breeze API to update
API DI services.

### CODE REVIEW

File: Stratis.Bitcoin/RPC/RPCFeature.cs [hunk 1]
```diff
@@ -58,7 +78,8 @@ public static IFullNodeBuilder AddRPC(this IFullNodeBuilder fullNodeBuilder)
             fullNodeBuilder.ConfigureFeature(features =>
             {
                 features
-                .AddFeature<RPCFeature>();
+                .AddFeature<RPCFeature>()
+                .FeatureServices(services => services.AddSingleton(fullNodeBuilder));
```
Comment (dangershony) [inline Stratis.Bitcoin/RPC/RPCFeature.cs]: hmm this could go in the base feature it occurred  to me but fine for now.

**Comment by dangershony** [comment]
This looks good the integration tests are passing?

**Comment by mikedennis** [comment]
Removed the manual fullnode DI configuration in RPC since commit should take care of that. Ran integration tests and they pass.

**Comment by NicolasDorier** [comment]
tested ACK 7c43f17 (The tests that needed my previous hack are still passing with this PR)

**Comment by dangershony** [comment]
Great job

**Pull request merged**